### PR TITLE
Enable metafix (#1345)

### PR DESCRIPTION
**Pull request by dr0i**
Provides an example of Alma Marc XML ETL (based on the input used by morph ETL).
Prerequisites:
install metafix locally;
- checkout https://github.com/metafacture/metafacture-fix.git
- change to that directory
- `./gradlew publishToMavenLocal`
This installs metafix to a local maven repo in: '~/.m2/repository/org/metafacture/metafix/'
The version 0.3.0-SNAPSHOT is needed.
- change directory to lobid-resources
Run like:
`mvn clean install -DskipTests=true ; mvn -DgenerateTestData=true failsafe:integration-test -Dit.test=AlmaMarc21XmlToLobidJsonMetafixTest.java`.

See #1345.

### CODE REVIEW

**Review by TobiasNx — APPROVED** [review]
+1 With respect  to the function I think I could start :) and I already did.

File: src/test/resources/alma-fix/(DE-605)HT000161712.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "almaMmsId" : "990001412590206441",
+  "@context" : "https://lobid.org/resources/context.jsonld",
+  "id" : "https://lobid.org/resources/990001412590206441#!https://lobid.org/resources/#!"
+}
```
[the remark this answers is not in the document]
Comment (dr0i) [reply src/test/resources/alma-fix/(DE-605)HT000161712.json:5]: there are two superfluous spaces

**Review by dr0i — COMMENTED** [review]
Looks good now +1

**Comment by TobiasNx** [comment]
Then +1 from me too. You could merge then.

**Comment by dr0i** [comment]
Merged. Closed.

**Pull request merged**